src/plugins/saved_search/server/saved_objects/search.ts

```diff
@@ -33,44 +35,74 @@ export function getSavedSearchObjectType(
       },
     },
     mappings: {
+      dynamic: false,
       properties: {
-        columns: { type: 'keyword', index: false, doc_values: false },
         description: { type: 'text' },
-        viewMode: { type: 'keyword', index: false, doc_values: false },
-        hideChart: { type: 'boolean', index: false, doc_values: false },
-        isTextBasedQuery: { type: 'boolean', index: false, doc_values: false },
-        usesAdHocDataView: { type: 'boolean', index: false, doc_values: false },
-        hideAggregatedPreview: { type: 'boolean', index: false, doc_values: false },
-        hits: { type: 'integer', index: false, doc_values: false },
-        kibanaSavedObjectMeta: {
-          properties: {
-            searchSourceJSON: { type: 'text', index: false },
-          },
-        },
-        sort: { type: 'keyword', index: false, doc_values: false },
         title: { type: 'text' },
-        grid: { dynamic: false, properties: {} },
         version: { type: 'integer' },
```

Comment:
Does anyone know what the `version` field is used for? It doesn't seem to exist in any of our models, but I was hesitant to drop an indexed field called `version`. If we don't think it's used anymore though, I'd be willing to drop it as well.

Reply:
Agree with dropping `version`. Probably `hits` too.

Reply:
`hits` was already dropped, but I ended up dropping `version` too and better organized the `schema`.